src/gpodder/schema.py

```diff
@@ -194,6 +199,9 @@ def upgrade(db, filename):
         initialize_database(db)
         return
 
+    db.create_function('is_html', 1, util.is_html)
+    db.create_function('remove_html_tags', 1, util.remove_html_tags)
```

Comment:
I looked for a way to delete these functions when we are done and could not find anything.  I am assuming they are transient.